includes/icon-functions.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * SVG icons related functions
+ *
+ * @package TenUpScaffold
+ */
+
+/**
+ * An svg template helper.
+ *
+ * Example:
+ * tenupscaffold_svg_icon(
+ *     'instagram', array(
+ *         'role'   => 'img',
+ *         'class'  => 'tenup-icon instagram-icon',
+ *         'label'  => 'Follow 10up on Instagram',
+ *         'hidden' => 'true',
+ *     ),
+ *     true,
+ * );
+ *
+ * @param  string  $name   The SVG Name.
+ * @param  array   $opts   Optional. The SVG options such as Role, Class, Label and Hidden.
+ * @param  boolean $output Either return or output the SVG Icon string.
+ * @return html
+ */
+function tenupscaffold_svg_icon( $name = '', $opts = array(), $output = false ) {
```

Comment:
@colorful-tones These functions should be namespaced with the rest of the utility functions in utility.php so the function call is `TenUpScaffold\Utility\svg_icon()` we don't want `tenupscaffold` right in the function name like that.

Reply:
@tim based on a recent conversation with @christianc1 

https://github.com/10up/theme-scaffold/issues/128#issuecomment-484736188

I modeled mostly what TwentyNineteen is doing here (see [TwentyNineteen's `includes/icon-functions.php`](https://github.com/WordPress/twentynineteen/blob/master/inc/icon-functions.php)) and would recommend we stay away from Namespacing these functions, because it is easier to drop them in to a template file without having to declare `use` them. 

They're mostly template tag style functions. Since there is enough code to warrant its own file then I did not include them in the existing (and recently emptied) `includes/template-tags.php` file. (Again, modeling TwentyNineteen in usage and perhaps standard expectation)

I  may be misunderstanding the intent here and would appreciate input from @christianc1  as well. 

Thanks!